joinmarket-webui/jam
# feat: abort collaborative transaction

Closes #269.

Adds the ability to abort a running taker operation.

<img src="https://user-images.githubusercontent.com/3358649/190448985-6fbce2ae-02e0-44e8-91f9-c0f36aeb2d13.png" width=300 />
<img src="https://user-images.githubusercontent.com/3358649/190449063-42f755ee-9ca3-4da1-a3f2-6d060456ab68.png" width=300 />


### How to test
1. Send a collaborative transaction
2. Try to abort the transaction
3. Verify that the transaction was aborted and not broadcasted

You might want to watch the logs while testing this feature with `npm run regtest:logs:jmwalletd`

Question: Currently, the text only says "Are you sure you want to abort the collaborative transaction?".. 
Stopping a taker operation might lead to end up wasting a "retry" (when commitment was already sourced).. Should it mention that it might lead to inconveniences if this is used "too much in a row"? How to communicate this best?
